src/models/discreteBar.js

```diff
@@ -49,6 +51,14 @@ nv.models.discreteBar = function() {
                 series.values.forEach(function(point) {
                     point.series = i;
                 });
+
+                // Changes the availableWidth to allow for a minimum bar width
+                if(minBarWidth) {
+                    var minWidth = series.values.length * minBarWidth * 1.1;
+                    if(availableWidth < minWidth) {
+                        availableWidth = minWidth;
+                    }
+                }
```

Comment:
Oh wait, I see what this is doing...  but it should still not change availableWidth I don't think...   since the bar charts currently make the bars as thick as they can be given the width, why would you ever want to force them to be bigger like this?